File: demo-iys-rest-scraping.java/src/main/java/demo/delay/Delay.java
package demo.delay;

public final class Delay {

    public static void seconds(int seconds) {
        while (seconds-- > 0) {
            second();
        }
    }

    private static void second() {
        try {
            Thread.sleep(1000);
        } catch (InterruptedException e) {
            throw new RuntimeException(e);
        }
    }
}

File: demo-iys-rest-scraping.kotlin/src/main/kotlin/demo/delay/Delay.kt
package demo.delay

object Delay {

    fun seconds(seconds: Int) {
        var seconds = seconds
        while (seconds-- > 0) {
            second()
        }
    }

    private fun second() {
        try {
            Thread.sleep(1000)
        } catch (e: InterruptedException) {
            throw RuntimeException(e)
        }

    }
}
